homeassistant/components/ovo_energy/sensor.py

```diff
@@ -19,6 +32,13 @@
 PARALLEL_UPDATES = 4
 
 
+@dataclass
+class OVOEnergySensorEntityDescription(SensorEntityDescription):
+    """Class describing System Bridge sensor entities."""
+
+    value: Callable = round
```

Comment:
You can add additional type information to `Callable`.
I think here it's probably: `Callable[[OVODailyUsage], StateType]`.